src/formatter.rs

```diff
@@ -0,0 +1,129 @@
+// Custom serde_json formatter supporting ReJSON formatting options.
+
```

Comment:
This looks like a fork of `serde_json::ser::PrettyFormatter` -- which is fine of course.

Please mention that in a comment, for future reference -- and also for licensing reasons (this file should probably be licensed as MIT/Apache like `serde_json` itself).

Reply:
Done.